database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,10 +15,14 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->bigIncrements('id');
-            $table->string('name');
+            $table->string('first_name');
+            $table->string('last_name');
+            $table->string('username');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
+            $table->binary('avatar');
```

Comment:
set default value because when I register new user it will return an error
[![Image from Gyazo](https://i.gyazo.com/98f682ab9fdf0ed33adb78e2fd0e5621.png)](https://gyazo.com/98f682ab9fdf0ed33adb78e2fd0e5621)